### ISSUE TITLE
pandoc url is wrong for R packages on OS X 

### ISSUE THREAD

**Issue by dongzhuoer** [travis-ci/docs-travis-ci-com#1825]
at least for 2.1.2, the url should be https://github.com/jgm/pandoc/releases/download/2.1.2/pandoc-2.1.2-macOS.pkg instead of https://github.com/jgm/pandoc/releases/download/2.1.2/pandoc-2.1.2-osx.pkg

https://travis-ci.org/dongzhuoer/zhuoerdown/jobs/355689693#0-69

@craigcitro 
@jimhester 

see travis-ci/travis-ci#9367

**Comment by dongzhuoer** [comment]
For users, you may exclude OS X

```yaml
os: linux
```

or allow it to fail
```yaml
os: [linux, osx]
matrix:
  allow_failures:
  - os: osx
```

**Comment by coatless** [comment]
Addressed in travis-ci/travis-build#1373 and travis-ci/docs-travis-ci-com#1827. This can be closed out.
